src/modules/Makefile.am

```diff
@@ -25,3 +25,6 @@ urpc_la_LIBADD += $(PERFFLOW_LIBS)
 urpc_la_CPPFLAGS += $(PERFFLOW_PLUGIN_CPPFLAGS) $(PERFFLOW_CFLAGS) -DDYAD_PERFFLOW=1
 endif
 endif
+
+install-exec-hook:
+	@(cd $(DESTDIR)$(libdir) && $(RM) $(lib_LTLIBRARIES) dyad.a)
```

Comment:
I think it's fine to use `$(lib_LTLIBRARIES)` here, but it might be more clear to use `dyad.la` instead.

Reply:
making that change.